docs/reference/commands/pack.rst

```diff
@@ -0,0 +1,48 @@
+
+.. _ref_commands_pack:
+
+pack
+====
+
+Process parts and create the final artifact.
```

Comment:
Should we maybe be a bit more specific about what this means? Since right now it's rather quite generic. I think it would be best if we at least said that at this stage, in most cases, this is when the 'image' in image-craft is created - usually a disk with partitions. Of course in later imagecraft versions we might support other formats, like tarballs or something else crazy.

Was thinking about maybe we should also mention that `ubuntu-image pack` is used for this stage? I know it's an implementation detail, but for people that are encountering errors that might be a useful fact. I'm not very strongly opinionated about this though.

Reply:
Every file in `docs/reference/commands/` is generated. Since we do not define any specific command, all of this comes from some craft libs. However your suggestions should probably go into the `explanation` section or be part of the tutorial.